README.md

```diff
@@ -32,22 +32,24 @@ Locally these services can be run in two modes:
 #### Dependencies and Installation
 
 Local development requires that the following services be available. They do not need to be running locally, but for development purposes this is probably easiest. These should be installed by whatever means is easiest (on macOS this is generally `brew`, or your package manager of choice). These dependencies are:
-- PostgreSQL@10
-- ElasticSearch@7.10>
+- PostgreSQL@10 
+  - Note that v10 is deprecated.
+- ElasticSearch@7.10 
+  - Note you may need to follow the [macOS Homebrew install guide](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/brew.html#brew).
 - RabbitMQ
 - Redis
 - XCode Command Line Tools
 
-This is a Python application and requires Python >= 3.6. It is recommended that a virtual environment be set up for the application (again use the virtual environment tool of your choice).
+This is a Python application and requires Python >= 3.6. It is recommended that a virtual environment be set up for the application (again use the virtual environment tool of your choice).  There are several options, but most developers use [virtualenv](https://virtualenv.pypa.io/en/latest/installation.html#).
```

Comment:
It's true that virtualenv is probably still more widely used, and I don't object to this addition, but [the Python-maintained `venv` module](https://docs.python.org/3/library/venv.html) introduced in 3.3 (and recommended as of 3.5) does seem to work for this repo and I've used it successfully for all of DRB and PJR's repositories without installing another virtual environment manager.

Reply:
I'll add it to the recommendations!